app_flutter/lib/service/appengine_cocoon.dart

```diff
@@ -6,12 +6,10 @@ import 'dart:convert';
 import 'dart:io';
 
 import 'package:app_flutter/logic/qualified_task.dart';
+import 'package:cocoon_service/protos.dart';
```

Comment:
The import changes do not conform with the style guide: https://github.com/flutter/cocoon/tree/master/app_flutter#imports